Patterns/Dutch/Dutch-Numbers.yaml

```diff
@@ -205,7 +205,7 @@ HalfADozenText: zes
 WordSeparatorToken: en
 WrittenDecimalSeparatorTexts: [komma]
 WrittenGroupSeparatorTexts: [punt]
-WrittenIntegerSeparatorTexts: [en]
+WrittenIntegerSeparatorTexts: [en, ën]
```

Comment:
Are there tests in the DutchNumber parser for this?

Reply:
yes, for example "tweeënvijftig" and "tweeenvijftig" (fifty two) are both included in NumberModel.json